java-logger/src/main/resources/archetype-resources/src/main/resources/my-logger.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "streamzi.io/v1"
+kind: Processor
+metadata:
+  name: log-data
+spec:
+  displayName: "Log Data"
+  description: "Prints the data received to console"
+  imageName: "${dhOrg}/${artifactId}"
```

Comment:
I've put it in `src/main/resources` - seemed best from a first, quick look

Reply:
👍